### PR TITLE
Migration guide for `toggleableActiveColor`

### PR DESCRIPTION
**Pull request by TahaTesser**
Migration guide for https://github.com/flutter/flutter/pull/95870

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by domesticmouse** [comment]
Looks like you broke the linter somehow: https://github.com/flutter/website/runs/4692946226?check_suite_focus=true

File: src/release/breaking-changes/index.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ release, and listed in alphabetical order:
 [Required Kotlin version]: {{site.url}}/release/breaking-changes/kotlin-version
 [Deprecated API removed after v2.5]: {{site.url}}/release/breaking-changes/2-5-deprecations
 [Raw images on Web uses correct origin and colors]: {{site.url}}/release/breaking-changes/raw-images-on-web-uses-correct-origin-and-colors
+[ThemeData's toggleableActiveColor property have been deprecated]: {{site.url}}/release/breaking-changes/toggleable-active-color
```
Comment (guidezpl) [inline src/release/breaking-changes/index.md:22]:
```suggestion
[ThemeData's toggleableActiveColor property has been deprecated]: {{site.url}}/release/breaking-changes/toggleable-active-color
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: ThemeData's toggleableActiveColor property have been deprecated
```
Comment (guidezpl) [inline src/release/breaking-changes/toggleable-active-color.md:2]:
```suggestion
title: ThemeData's toggleableActiveColor property has been deprecated
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+description: Material Widgets that use toggleableActiveColor property are migrated to use Material ColorScheme.
+---
+
+## Summary
+
+The widgets such as Switch, `SwitchListTile`, `Checkbox`, `CheckboxListTile`, `Radio`, `RadioListTile` have been adopted to use `ColorScheme.secondary` color for its toggleable widget. `ThemeData.toggleableActiveColor` is deprecated and will eventually be removed.
```
Comment (guidezpl) [inline src/release/breaking-changes/toggleable-active-color.md:8]:
```suggestion
The Material widgets `Switch`, `SwitchListTile`, `Checkbox`, `CheckboxListTile`, `Radio`, `RadioListTile` now use `ColorScheme.secondary` color for their toggleable widget. `ThemeData.toggleableActiveColor` is deprecated and will eventually be removed.
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+## Timeline
+
+Landed in version: 2.9.0-1.0.pre.209<br>
+In stable release: not yet
+
+## References
+
+API documentation:
+
+* [`toggleableActiveColor`][]
```
Comment (guidezpl) [inline src/release/breaking-changes/toggleable-active-color.md:54]:
```suggestion
* [`ThemeData.toggleableActiveColor`][]
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+Migration of widget that depends on `ThemeData.toggleableActiveColor` to `ColorScheme.secondary` rendered toggleableActiveColor unused which caused the deprecationn of this theme property.
+
+## Description of change
+
+The widgets using `ThemeData.toggleableActiveColor` color for selected state now uses `ColorScheme.secondary` color.
+
+## Migration guide
+
+In order to customize toggleable widgets selected color, remnove `ThemeData.toggleableActiveColor` and customize `ColorScheme.secondary` instead.
```
Comment (guidezpl) [inline src/release/breaking-changes/toggleable-active-color.md:20]:
```suggestion
In order to customize toggleable widgets' selected color, remove `ThemeData.toggleableActiveColor` and customize `ColorScheme.secondary` instead.
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+The widgets such as Switch, `SwitchListTile`, `Checkbox`, `CheckboxListTile`, `Radio`, `RadioListTile` have been adopted to use `ColorScheme.secondary` color for its toggleable widget. `ThemeData.toggleableActiveColor` is deprecated and will eventually be removed.
+
+## Context
+
+Migration of widget that depends on `ThemeData.toggleableActiveColor` to `ColorScheme.secondary` rendered toggleableActiveColor unused which caused the deprecationn of this theme property.
+
+## Description of change
+
+The widgets using `ThemeData.toggleableActiveColor` color for selected state now uses `ColorScheme.secondary` color.
```
Comment (guidezpl) [inline src/release/breaking-changes/toggleable-active-color.md:16]:
```suggestion
The widgets using `ThemeData.toggleableActiveColor` color for the selected state now use `ColorScheme.secondary`.
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+The Material widgets `Switch`, `SwitchListTile`, `Checkbox`, `CheckboxListTile`, `Radio`, `RadioListTile` now use `ColorScheme.secondary` color for their toggleable widget. `ThemeData.toggleableActiveColor` is deprecated and will eventually be removed.
+
+## Context
+
+Migration of widget that depends on `ThemeData.toggleableActiveColor` to `ColorScheme.secondary` rendered toggleableActiveColor unused which caused the deprecationn of this theme property.
+
+## Description of change
+
+The widgets using `ThemeData.toggleableActiveColor` color for the selected state now use `ColorScheme.secondary`.
```
Comment (guidezpl) [inline src/release/breaking-changes/toggleable-active-color.md:16]:
```suggestion
The widgets using `ThemeData.toggleableActiveColor` color for the active/selected state now use `ColorScheme.secondary`.
```

**Review by guidezpl — APPROVED** [review]
a few more comments, but LGTM

**Comment by TahaTesser** [comment]
@guidezpl 
Appreciate your review!

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
PTAL @sfshaza2

File: src/release/breaking-changes/toggleable-active-color.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+description: Material Widgets that use toggleableActiveColor property are migrated to use Material ColorScheme.
+---
+
+## Summary
+
+The Material widgets `Switch`, `SwitchListTile`, `Checkbox`, `CheckboxListTile`, `Radio`, `RadioListTile` now use `ColorScheme.secondary` color for their toggleable widget. `ThemeData.toggleableActiveColor` is deprecated and will eventually be removed.
+
+## Context
+
+Migration of widgets that depends on `ThemeData.toggleableActiveColor` to `ColorScheme.secondary` rendered toggleableActiveColor unuse, hence the deprecationn of this theme property.
```
Comment (sfshaza2) [inline src/release/breaking-changes/toggleable-active-color.md:12]:
```suggestion
The migration of widgets that depend on `ThemeData.toggleableActiveColor`
to `ColorScheme.secondary` caused  the `toggleableActiveColor` property
to be unnecessary. This property will eventually be removed, as per Flutter's
[deprecation policy]({{site.url}}/resources/compatibility#deprecation-policy).
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
+---
+title: ThemeData's toggleableActiveColor property has been deprecated
+description: Material Widgets that use toggleableActiveColor property are migrated to use Material ColorScheme.
+---
+
+## Summary
+
+The Material widgets `Switch`, `SwitchListTile`, `Checkbox`, `CheckboxListTile`, `Radio`, `RadioListTile` now use `ColorScheme.secondary` color for their toggleable widget. `ThemeData.toggleableActiveColor` is deprecated and will eventually be removed.
```
Comment (sfshaza2) [inline src/release/breaking-changes/toggleable-active-color.md:8]:
```suggestion
The Material widgets `Switch`, `SwitchListTile`, `Checkbox`,
`CheckboxListTile`, `Radio`, and `RadioListTile` now use
the `ColorScheme.secondary` color for their toggleable widget.
The `ThemeData.toggleableActiveColor` property is deprecated and will eventually be removed.
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+## Context
+
+Migration of widgets that depends on `ThemeData.toggleableActiveColor` to `ColorScheme.secondary` rendered toggleableActiveColor unuse, hence the deprecationn of this theme property.
+
+## Description of change
+
+The widgets using `ThemeData.toggleableActiveColor` color for the active/selected state now use `ColorScheme.secondary`.
```
Comment (sfshaza2) [inline src/release/breaking-changes/toggleable-active-color.md:16]:
```suggestion
The widgets using `ThemeData.toggleableActiveColor` color for the
active/selected state now use `ColorScheme.secondary`.
```

File: src/release/breaking-changes/toggleable-active-color.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+
+## Migration guide
+
+Toggleable widgets' active/selected color can generally customized in 3 ways at the widget-level using components themes `SwitchThemeData`, `ListTileThemeData`, `CheckboxThemeData`, and `RadioThemeData` using `ColorScheme.secondary`.
```
Comment (sfshaza2) [inline src/release/breaking-changes/toggleable-active-color.md:20]: @TahaTesser, I'm struggling with the following sentence. What are the 3 ways
you can customize the toggleable widget's active color? You list 4 ThemeData
classes and the ColorScheme class. So, I count either 5 ways, or 2 ways,
depending on how you count. Not 3. Can you clarify?
```suggestion
Toggleable widgets' active/selected color can generally customized in 3 ways
at the widget level using components themes `SwitchThemeData`, `ListTileThemeData`,
`CheckboxThemeData`, and `RadioThemeData` using `ColorScheme.secondary`.
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Thanks, @TahaTesser, and sorry for the delay in reviewing.

File: src/release/breaking-changes/toggleable-active-color.md (hunk 11 above)
Comment (TahaTesser) [reply src/release/breaking-changes/toggleable-active-color.md]: Thanks so much, I realized that it wasn't clear. I just updated this section.

**Review by sfshaza2 — APPROVED** [review]
LGTM! Thx!

**Pull request merged**